relnotes/chainsaw.deprecation.md

```diff
@@ -0,0 +1,8 @@
+* `ocean.util.log.LayoutChainsaw`
+
+  This module, and the eponymous class it contains, are not used in any project,
+  and thus were deprecated.
+  As a result, it is not configurable by the `LogExt` anymore.
+  If really needed, this self contained module can trivially be copied,
+  and configuration is possible via the `makeLayout` parameter in
+  `configure{Old,New}Loggers`.
```

Comment:
Please say here too why was it deprecated. As a user that was using the module, reading the release notes it will be my first question. "But why?!?!?" :)